models/intel/action-recognition-0001/description/action-recognition-0001.md

```diff
@@ -0,0 +1,68 @@
+# Composite model name: action-recognition-0001
+
+## Use Case and High-Level Description
+
+This is a general-purpose action recognition composite model, consisting from encoder and decoder parts, trained on Kinetics-400 dataset. The encoder model uses Video Transformer approach with ResNet34 encoder.
+Please refer to the [kinetics](https://deepmind.com/research/open-source/open-source-datasets/kinetics/) dataset specification to see list of action that are recognised by this composite model.
+
+## Example
+
+![](./demo.png)
+
+## Encoder model specification
+
+The [encoder model](../action-recognition-0001-encoder) accepts video frame and produces embedding.
+Video frames should be sampled to cover ~1 second fragment (i.e. skip every second frame in 30 fps video).
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Source framework                | PyTorch*                                  |
+| GFlops                          | 7.340                                     |
+| MParams                         | 21.276                                    |
+
+
+## Performance
+
+## Inputs
+
+1. name: "0", shape: [1x3x224x224] - An input image in the format [BxCxHxW],
+   where:
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order is BGR.
+
+## Outputs
+
+The model outputs a tensor with the shape [1x512x1x1], representing embedding of processed frame.
+
+
+## Decoder model specification
+
+The [decoder model](../action-recognition-0001-decoder) accepts stack of frame embeddings, computed by [encoder model](../action-recognition-0001-encoder).
```

Comment:
You shouldn't link to folders, because there are no folders in the online docs. I think in this case it's not necessary to link the model names at all.

Reply:
ok, removed